contracts/token/erc20/preset/ImmutableERC20.sol

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Immutable Pty Ltd 2018 - 2024
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.19;
+
+import {ERC20Permit, ERC20} from "@openzeppelin/contracts/token/ERC20/extensions/ERC20Permit.sol";
+import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
+import {MintingAccessControl} from "../../../access/MintingAccessControl.sol";
+import {IImmutableERC20Errors} from "../../../errors/ERC20Errors.sol";
+
+
+
+/**
+ * @notice ERC 20 contract that wraps Open Zeppelin's ERC 20 contract.
+ * This contract has the concept of an owner, called _hubOwner in the constructor.
+ * This account has no rights to execute any administrative actions within the contract,
+ *  with the exception of transferOwnership. This account is accessed via the owner() 
+ *  function. The Immutable Hub uses this function to help associate the ERC 20 contract 
+ *  with a specific Immutable Hub account.
+ */
+contract ImmutableERC20 is Ownable, ERC20Permit, MintingAccessControl {
+    /**
+     * @dev Delegate to Open Zeppelin's contract.
+     * @param _name  Name of the token.
+     * @param _symbol Token symbol.
+     * @param _hubOwner The account associated with Immutable Hub.
+     * @param minterRole The account that has the MINTER_ROLE.
+     */
+    constructor(string memory _name, string memory _symbol, address _hubOwner, address minterRole) ERC20(_name, _symbol) ERC20Permit(_name) {
```

Comment:
I don't think this variable needs to be hub aware: `_hubOwner`. It can be called `_owner` instead because this can be deployed outside hub too.